src/models/supplemental_accessors.jl

```diff
@@ -23,3 +23,15 @@ function get_series_susceptance(b::PhaseShiftingTransformer)
 end
 
 get_series_admittance(b::ACBranch) = 1 / (get_r(b) + get_x(b) * 1im)
+
+function get_max_active_power(d::Device)
+    return get_active_power_limits(d).max
+end
+
+function get_max_reactive_power(d::Device)
+    return get_reactive_power_limits(d).max
+end
+
+function get_max_active_power(d::RenewableGen)
```

Comment:
Consider putting this in the same file where RenewableGen is defined.  Consider also implementing default methods for get_rating and get_power_factor that call `error("not implemented")`.  Or add a comment next to RenewableGen that says those methods are required.

Reply:
I've made the updates in c5c1d2f and 8139779